android/src/app/organicmaps/routing/NavigationController.java

```diff
@@ -214,10 +214,10 @@ public void update(@Nullable RoutingInfo info)
 
   private void updateStreetView(@NonNull RoutingInfo info)
   {
-    boolean hasStreet = !TextUtils.isEmpty(info.nextStreet);
-    UiUtils.showIf(hasStreet, mStreetFrame);
-    if (!TextUtils.isEmpty(info.nextStreet))
-      mNextStreet.setText(info.nextStreet);
+    mNextStreet.setText(info.nextStreet);
```

Comment:
If the fix is about showId -> visibleIf, should we keep setText under if not isEmpty?

Reply:
Yes, we can. I just wanted to streamline the code.

Reply:
@vng keep this refactored version or minimize the patch?

Reply:
Hm, better to leave under if, I think. What if info.nextStreet is null?

Reply:
ok, no brainer.